src/test/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentServiceSpec.scala

```diff
@@ -32,16 +32,19 @@ class ConsignmentServiceSpec extends AnyFlatSpec with MockitoSugar with ResetMoc
 
```

Comment:
Not related to your changes but there are a couple of unused imports that should be removed:

- `import uk.gov.nationalarchives.tdr.api.graphql.fields.{..., SeriesFields}`
- `import java.time.Instant.now`